src/scripts/components/VariableMessage.jsx

```diff
@@ -82,14 +82,24 @@ class VariableMessage extends React.Component {
   transformVar = (text) => {
     // Re-assign text value for processing
     let value = text;
-
-    // Replace text contents of variable to hide squigglies
-    const regexSquiggles = /{(.*)?}/g;
-    value = value.replace(regexSquiggles, '<i>{</i>$1<i>}</i>');
-
-    // Replace text contents of variable to hide underscore
-    const regexUnderscores = /_/g;
-    value = value.replace(regexUnderscores, '<b>_</b>');
+    // We are treating these as a special case because formatting for the "to" needs to me lowercase
+    if (text === '{REPLY_‘1’_to_CONFIRM}' || text === '{REPLY_‘2’_to_CANCEL}') {
```

Comment:
I wonder if @alexgsummer has any suggestions for this, I don't really love the idea of having hard coded logic in rhinostyle for specific variable messages.